torch/_torch_docs.py

```diff
@@ -7453,6 +7453,34 @@ def merge_dicts(*dicts):
     tensor([-0.5194,  0.1343, -0.4032, -0.2711])
 """.format(**common_args))
 
+add_docstr(torch.sinc,
+           r"""
+sinc(input, *, out=None) -> Tensor
+
+Returns a new tensor with the normalized sinc of the elements of :attr:`input`.
```

Comment:
"Returns a new tensor with the normalized sinc..." -> "Computes the normalized sinc of :attr:\`input\.`